### DISCUSSION TITLE
NotificationBlock - A bit difficult to find?

### DISCUSSION THREAD

**Discussion by paints-with-bold-colours** [AprilSylph/XKit-Rewritten#149] [Suggestion Box]
Hello!
I was very happy to see NotificationBlock added to Rewritten - thank you for adding it!
It did take me quite a bit to figure out _how_ to block notifcations on a post; I'm used to easily finding the button from new xkit. It's only thanks to the Wiki here that I figured out it was in the meatballs menu - which I never use and so forgot existed lol.
I don't know if other users will have the same trouble as me finding it, but I figured they might? And I thought this could be solved by specifying.
So the description for NotificationBlock could maybe say "Block a post's notifcations _via its meatballs menu_"? I realise this is longer than the other descriptions and might create spacing/layout issues, so maybe there's another solution - or maybe this isn't even necessarily a problem that needs a solution - but I figured I'd throw it out there.

**Comment by AprilSylph** [comment]
Perhaps as more scripts add items to the meatballs menu, it will become natural to look for them there. I will keep this in mind, though; while the wiki is designed to serve as a manual for the software, it really would be best to eliminate the need for one. Perhaps I will end up implementing longer collapsible descriptions for each script after all.
